dashboard/app.py

```diff
@@ -77,25 +84,39 @@
                 'fonts.googleapis.com',
                 'https://*.googletagmanager.com',
                 'https://tagmanager.google.com',
-                'https://*.google-analytics.com'
+                'https://*.google-analytics.com',
+                'https://cdn.sso.mozilla.com',
+                'https://cdn.sso.allizom.org',
+                'https://dhjrqi6qcwjfu.cloudfront.net'
```

Comment:
Is this a different distribution to cdn.? If not, do we need both the cloudfront.net and mozilla.com urls?